lua/astrocommunity/pack/java/java.lua

```diff
@@ -1,5 +1,8 @@
 local utils = require "astrocommunity.utils"
 
+-- 5.1 lua compatibility
+table.unpack = table.unpack or unpack
+
```

Comment:
```suggestion
```

This causes Linting issues and we shouldn't override this function globally

Reply:
makes sense. I will update